KNOWN_ISSUES.md

```diff
@@ -1,14 +1,17 @@
 # v0.1.0
-* IP conflicts: By default, Cloud Filestore creation will pick an unused IP subnet to allocate its
+* IP conflicts: By default, Cloud Filestore creation will pick an unused IP range to allocate its
```

Comment:
I'm not sure why this diff is showing up because it's already like this in master.  Maybe you need to rebase?